README.rst

```diff
@@ -28,6 +28,16 @@ Dwarf-Therapist requires a C++ compiler (with C++14 support), cmake (3.1.0 or ne
 
 Detailed building instructions can be found in `BUILDING.md`_.
 
+Compatibility for latest DF version:
+====================================
+
+If the latest release is not yet available in the download page, it's possible that the necessary data (memory maps) are already updated in the repository.
+
+To update manually you can either:
+
+- build from source code yourself, as per the section above;
```

Comment:
Building from source for updating memory layouts is useless and complex. I would not recommend that in the readme.